systemtest/src/main/java/io/strimzi/systemtest/utils/kubeUtils/objects/SecretUtils.java

```diff
@@ -136,4 +137,14 @@ public static void waitForCertToChange(String originalCert, String secretName) {
             }
         });
     }
+
+    public static void deleteSecretWithWait(String secretName, String namespace) {
```

Comment:
Do we rly need this wait? I just didn't had any issues with delete secret

Reply:
Maybe we don't, I tried to remove some race conditions by adding this wait. But I can remove it :) thanks for the comment

Reply:
We discussed this offline -> we will keep this method, without it there is race condition -> creating secret when is in terminating state